pkg/util/k8sutil.go

```diff
@@ -82,6 +82,7 @@ func PodAntiAffinity(component string, clusterName string) *corev1.Affinity {
 // Wait for pods in cluster to be terminated
 func WaitForClusterToTerminate(kubeClient client.Client, p *v1alpha1.BookkeeperCluster) (err error) {
 	listOptions := &client.ListOptions{
+		Namespace:     p.Namespace,
```

Comment:
@SrishT could you please explain the root cause for this issue? If the zookeeper URI provided in the BK Cluster Spec is incorrect or not resolvable/reachable, BK cluster won;t be able to come up.... so why does this URI work in case of BK cluster and not work when invoked from BK Operator?

Reply:
Both the bookkeeper cluster and the zookeeper cluster were present in a different namespace, which was different from the namespace the operator was deployed in.
Hence the Zookeeper URI was reachable from the bookkeeper cluster but not from the operator.